pennylane/transforms/resource_estimation.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Code for resource estimation"""
+
+
+def resource_estimation(qnode, expand_depth=1):
```

Comment:
Hmm... not sure how I feel about `expand_depth` being here. But that leads into a whole other conversation around how tapes get expanded on the device, and that's probably outside of the design scope here....